runtime/core/decoder/torch_asr_decoder.h

```diff
@@ -24,6 +24,7 @@ using TorchModule = torch::jit::script::Module;
 struct DecodeOptions {
   int chunk_size = 16;
   int num_left_chunks = -1;
+  bool bpe_decode = true;
```

Comment:
bpe_decode is not required here, we also use ▁ for white space in character based training.